### PR TITLE
4 get api reviews

### PR DESCRIPTION
**Pull request by uzair-haroon**

### CODE REVIEW

File: models/games-model.js [hunk 1]
```diff
@@ -13,3 +13,19 @@ exports.selectCategories = () => {
             return result.rows;
         });
 };
+
+exports.selectReviews = () => {
+    return db
+        .query(
+            `
+                SELECT reviews.*, CAST(COUNT( comments.comment_id ) AS int) AS comment_count FROM reviews
+                INNER JOIN comments
```
Comment (dav3rid) [inline models/games-model.js]: Would opt for an alternative JOIN here to include any articles that do not have any associated articles

File: __tests__/app.test.js [hunk 2]
```diff
@@ -40,3 +40,32 @@ describe("/api/categories", () => {
...
+                const reviews = res.body.reviews;
+                expect(reviews.length).toBeGreaterThan(0);
+                reviews.forEach((review) => {
+                    expect(review).toMatchObject({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        category: expect.any(String),
+                        designer: expect.any(String),
+                        owner: expect.any(String),
+                        review_body: expect.any(String),
```
Comment (dav3rid) [inline __tests__/app.test.js]: We shouldn't have a review body present on the responses here - you may (definitely will!) need to amend your model to reflect

**Review by dav3rid — CHANGES_REQUESTED** [review]
A couple of minor alterations Uzair - I'm also a little concerned that the commits detailing the tests come after your implementations 🤔

**Comment by uzair-haroon** [comment]
> A couple of minor alterations Uzair - I'm also a little concerned that the commits detailing the tests come after your implementations 🤔

Hi Dave, It was a commit mistake. I did the implementation in a TDD fashion, but I forgot to commit the test file with the controller and model code, so I had to do an extra commit after to include the tests.

File: app.js [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 
-const { getCategories } = require("./controllers/games-controllers");
+const {
+    getCategories,
+    getReviews,
+} = require("./controllers/games-controllers");
 
 const app = express();
 
 app.use(express.json());
```
Comment (xViolaine) [inline app.js]: We do not need this line of code yet, so it should not be here

**Review by xViolaine — APPROVED** [review]
Looks good, nice implementation of the former feedback - just make sure to delete the "app.use(express.json());" line, as this will not become relevant until a later point 👀

**Pull request merged**